src/main/webapp/dev/js/common/datepicker.js

```diff
@@ -73,11 +75,24 @@ function prepareDatepickers() {
         showOtherMonths: true,
         gotoCurrent: true,
         defaultDate: yesterday,
-        maxDate: today,
+        maxDate: defaultStartDate,
```

Comment:
the reason i changed this part from `$('#startdate').val()` in my previous commit is that this function `prepareDatepickers()` are used in two places, `instructorFeedback.js`(adding a new session) and `instructorFeedbackEdit.js`(editing an existing session). The previous way works fine only when the `value` attribute of `$('#startdate')` has a proper date value. However this only happens when editing an existing session. When adding a new session, the `value` attribute of `$('#startdate')` has an empty string as the value. Since when adding a new session, the start date by default is set to `today`, I just set the `maxDate` of visible date to `today`. The `maxDate` range will change correspondingly when the start date is edited (as long as the datepicker for start date pops up i.e. `onSelect`).

Reply:
Does `$('#startdate').datepicker('getDate')` work?